From ed94679d7ab76970bc8b699d0b86e3e3869942cf Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Mon, 16 Mar 2020 17:33:46 +0200
Subject: [PATCH 081/269] s32g: Update license text

S32G eDMA header file had the wrong license; updated to BSD-v3.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-4756
---
 plat/s32g/include/s32g274a_edma.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/s32g/include/s32g274a_edma.h b/plat/s32g/include/s32g274a_edma.h
index 9684877ed..ab547ee32 100644
--- a/plat/s32g/include/s32g274a_edma.h
+++ b/plat/s32g/include/s32g274a_edma.h
@@ -1,7 +1,7 @@
 /*
  * Copyright 2020 NXP
  *
- * SPDX-License-Identifier:	GPL-2.0+
+ * SPDX-License-Identifier: BSD-3-Clause
  */
 
 #ifndef S32G274A_EDMA_H
-- 
2.17.1

